src/i18n/locales/en.ts

```diff
@@ -113,7 +113,7 @@ export class en implements Locale {
     return ", only on %s";
   }
   commaAndOnX0() {
-    return ", and on %s";
+    return ", or on %s";
```

Comment:
Thanks!

This is failing some tests and after reviewing the failures I do think that, at least for English, usage of the word 
"and" and clearer and more accurate.  For example:

`0 2,16 1,8,15,22 * 1,2`

This cron will only execute on on Monday, Tuesdays, certain days of the week, at 2 AM and 4 PM.  If we use the word "or" it could imply that it would execute every Monday and Tuesday _as well as_ on day 1, 8, 15, and 22 of the month.

> At 02:00 AM and 04:00 PM, on day 1, 8, 15, and 22 of the month, **and** on Monday and Tuesday

vs 

> At 02:00 AM and 04:00 PM, on day 1, 8, 15, and 22 of the month, **or** on Monday and Tuesday

This is a bit tricky but I think I'd prefer to leave the English translation in place.  If changing the Chinese translation makes more sense here that is fine.

Can you revert changes to `src/i18n/locales/en.ts`?  If so, then I can approve this and cut a release.

Reply:
I mainly think the word "and" is something like "&" in logic. But since I am not an English expert, I changed it back.